utilities/git/hooks/pre-commit.sh

```diff
@@ -20,7 +20,7 @@ unset GIT_LITERAL_PATHSPECS
 
 #Change the delimiter used by 'for' so we can handle spaces in names
 IFS=$'\n'
-for FILE in $(git diff --cached --name-only "*.cpp" "*.h")
+for FILE in $(git diff --diff-filter=d --cached --name-only "*.cpp" "*.h")
```

Comment:
This is an entirely reasonable change, but should have been in a separate PR.

Reply:
I agree. Should I create a separate PR?

Reply:
Don't bother. Now that we've looked at it and discussed it there's no gain.